layouts/partials/css.html

```diff
@@ -14,6 +14,19 @@
     background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAUAAAAeCAYAAADkftS9AAAAIklEQVQoU2M4c+bMfxAGAgYYmwGrIIiDjrELjpo5aiZeMwF+yNnOs5KSvgAAAABJRU5ErkJggg==');
     cursor: col-resize;
 }
+
+@media (max-width: 768px) {
+  #sidebarnav,
+  #maindoc {
+    width: 100% !important;
```

Comment:
Not ideal. We can merge this, but it's best to avoid `!important` in CSS website styles.

Reply:
Okay, I have updated the JS file and removed `!important` in the new commit. Now, the Split.js splitter is not initialized on mobile screen sizes. Please review the changes.